src/main/resources/config.yml

```diff
@@ -246,6 +246,10 @@ old-fishing-knockback:
   # This is the delay in milliseconds in-between rod damage, so the player hit has time to fall back down
   hitCooldown: 1000
 
+old-fishing-rod-velocity:
```

Comment:
Please have a consistent naming scheme. The module should just be called `fishing-rod-velocity` in both the class and config.